Changes requested by gudpotato

Glitchtest51 opened the pull request.

Glitchtest51 committed:
Edit tag dupe1.1.2

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
Edit tag 1.2.1fe

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
Add note for BID

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
Add version history wiki page

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
Add disclaimer for MNF on version 1.3.0-1.4.1

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

gudpotato commented on messages.py:
```diff
@@ -1040,7 +1040,7 @@
 - [Bomb/Elemental Fruit "Dupe"](<https://youtu.be/BZID7B_99QY?t=21>)
 '''
-INFO_DUPE121TITLE = '''Here are some duplication glitches for 1.2.1 or 1.4.0:'''
+INFO_DUPE121TITLE = '''Here are some duplication glitches for 1.2.1-1.4.0:'''
```
I still gotta figure out how to effectively use GitHub for for now, I’ll just leave comments for fixes:
The latest version is 1.4.1, not 1.4.0

Glitchtest51 committed:
edit

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
lots of aliases

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
disclaimer on zuggle

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

Glitchtest51 committed:
edits

Co-authored-by: gudpotato <217290549+gudpotato@users.noreply.github.com>

gudpotato committed:
Update messages.py

Added SID video links and some aliases in the help messsages

gudpotato committed:
Update credits and transferalbum

Updated the usernames for (ex) testers and added links for transferring screenshots and screen captures for the Nintendo Switch 2.
